.github/workflows/build.yml

```diff
@@ -42,8 +42,8 @@ jobs:
           WORKSPACE_DIR: jenkins-agents/stock
           PKR_VAR_source_image_family: jenkins-stock-agent
           PKR_VAR_image_prefix: jenkins-gpu-agent
-          PKR_VAR_nvidia_driver_version: "460.80"
-          PKR_VAR_nvidia_driver_base_url: "https://us.download.nvidia.com/XFree86/Linux-x86_64"
+          PKR_VAR_nvidia_driver_version: "460.106.00"
```

Comment:
is there any documentation we can add here to explain why/how we picked this version?

Reply:
I went to Nvidia's website and filled in the requisite info (T4 GPU, CUDA 11.2, Linux X86), and was directed to the page [here](https://www.nvidia.com/content/DriverDownload-March2009/confirmation.php?url=/tesla/460.106.00/NVIDIA-Linux-x86_64-460.106.00.run&lang=us&type=Tesla). Furthermore, I've tested the install script for this version internally and it runs with Linux kernel version 5.13, which is what we want.